services/integration-content-repository/k8s/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: icr
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: icr
+    spec:
+      containers:
+        - name: icr
+          image: openintegrationhub/icr:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3001
+          env:
+            - name: MONGODB_URL
+              value: 'mongodb://icrdb.openintegrationhub.com'
+            - name: IAM_JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-oih-iam-dev
+                  key: 'jwtsecret'
+            - name: IAM_JWT_AUDIENCE
+              value: 'openintegrationhub.com'
+            - name: IAM_JWT_SECRET
```

Comment:
Lines 28 - 32 are the same as in 21-25, is that really necessary?

Reply:
fixed